Gemfile.norbnacl

```diff
@@ -1,10 +0,0 @@
-source 'https://rubygems.org'
```

Comment:
Wasn't sure what this was all about, but hopefully I can address whatever problems were arising which brought about the need for this in the first place in the ed25519 gem.

Reply:
The idea about .norbnacl Gemfile is that we could ran test with rbnacl and without rbnacl.
It was required on windows as we did not have rbnacl there.